.env.example

```
@@ -0,0 +1,24 @@
+PORT=3000
+
+# Security - CHANGE ALL THESE VALUES
+ADMIN_PASSWORD=your-secure-admin-password
+API_KEY=generate-a-long-random-api-key
+SESSION_SECRET=generate-a-long-random-session-secret
+
+# CORS Configuration (comma-separated origins)
+ALLOWED_ORIGINS=http://localhost:3000,https://yourdomain.com
+
+# Rate Limiting
+RATE_LIMIT_WINDOW_MS=900000  # 15 minutes in milliseconds
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)